templates/settings-admin.php

```diff
@@ -266,25 +266,21 @@ class="barchart"
 				<div class="infobox">
 					<div class="phpinfo-wrapper">
 						<p>
-							<?php p($l->t('Version').':'); ?>
+							<?php p($l->t('Version:')); ?>
 							<em id="phpVersion"><?php p($_['php']['version']); ?></em>
 						</p>
 						<p>
-							<?php p($l->t('Memory limit').':'); ?>
+							<?php p($l->t('Memory limit:')); ?>
 							<em id="phpMemLimit"><?php p($_['php']['memory_limit']); ?></em>
 						</p>
 						<p>
-							<?php p($l->t('Max execution time').':'); ?>
+							<?php p($l->t('Max execution time:')); ?>
 							<em id="phpMaxExecTime"><?php p($_['php']['max_execution_time']); ?></em>
 						</p>
 						<p>
-							<?php p($l->t('Upload max size').':'); ?>
+							<?php p($l->t('Upload max size:')); ?>
 							<em id="phpUploadMaxSize"><?php p($_['php']['upload_max_filesize']); ?></em>
 						</p>
-						<p>
-							<?php p($l->t('Extensions:')); ?>
-							<em id="phpExtensions"><?php p($_['php']['extensions'] !== null ? implode(', ', $_['php']['extensions']) : $l->t('Unable to list extensions')); ?></em>
-						</p>
```

Comment:
?

Reply:
Is there something wrong with this code?

Reply:
Why do you remove it?

Reply:
Looks suspicious to me, too. Code was introduced by https://github.com/nextcloud/serverinfo/pull/367/

Reply:
Oops. I didn't notice it. Sorry, I was working with an older version of the file. I have already brought back an earlier shift 😉.